antsibull_changelog/plugins.py

```diff
@@ -379,6 +406,9 @@ def load_plugins(paths: PathsConfig,
     :arg use_ansible_doc: Set to ``True`` to always use ansible-doc to enumerate plugins/modules
     :return: A list of all plugins
     """
+    if paths.is_other_project:
```

Comment:
In addition to my first comment, you could just check to see if this is a collection here instead, and return the empty array if it's not a collection?

Reply:
Same here, the function must run as before for ansible-core/-base and for collections, but it must be skipped for other projects.